src/zzz_od/operation/back_to_normal_world.py

```
@@ -58,25 +59,48 @@ def check_screen_and_run(self) -> OperationRoundResult:
-        # 这可以是通用的退出战斗 退出战斗的画面也有返回按钮 需要在返回前面
-        result = self.round_by_find_and_click_area(self.last_screenshot, '零号空洞-战斗', '退出战斗')
+
+        # 部分画面有关闭按钮 置前，插件场景"关闭"和"合成（完成）"可能同时存在
+        result = self.round_by_find_and_click_area(self.last_screenshot, '画面-通用', '关闭')
         if result.is_success:
-            self.click_exit_battle = True
             return self.round_retry(result.status, wait=1)
 
-        if self.click_exit_battle:
-            result = self.round_by_find_and_click_area(self.last_screenshot, '零号空洞-战斗', '退出战斗-确认')
+        # 战斗菜单-退出战斗（完全通用，包括但不限于危局强袭战！）
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)